hoomd/md/compute.py

```diff
@@ -282,84 +283,71 @@ def num_particles(self):
             return None
 
 
-class thermoHMA(Compute):
+class ThermoHMA(Compute):
     R""" Compute HMA thermodynamic properties of a group of particles.
 
     Args:
-        group (``hoomd.group``): Group to compute thermodynamic properties for.
+        filter (``hoomd.filter``): Particle filter to compute thermodynamic
+            properties for.
         temperature (float): Temperature
-        harmonicPressure (float): Harmonic contribution to the pressure.  If ommitted, the HMA pressure can still be
-            computed, but will be similar in precision to the conventional pressure.
-
-    :py:class:`thermoHMA` acts on a given group of particles and calculates HMA (harmonically mapped
-    averaging) properties of those particles when requested.  HMA computes properties more precisely (with less
-    variance) for atomic crystals in NVT simulations.  The presence of diffusion (vanacy hopping, etc.) will prevent
-    HMA from providing improvement.  HMA tracks displacements from the lattice positions, which are saved when the
-    :py:class:`thermoHMA` is instantiated.
-
-    The specified properties are available for logging via the ``hoomd.analyze.log`` command. Each one provides
-    a set of quantities for logging, suffixed with *_groupname*, so that values for different groups are differentiated
-    in the log file. The default :py:class:`thermoHMA` specified on the group of all particles has no suffix
-    placed on its quantity names.
-
-    The quantities provided are (where **groupname** is replaced with the name of the group):
-
-    * **potential_energyHMA_groupname** - :math:`U` HMA potential energy that the group contributes to the entire
-      system (in energy units)
-    * **pressureHMA_groupname** - :math:`P` HMA pressure that the group contributes to the entire
-      system (in pressure units)
-
-    See Also:
-        ``hoomd.analyze.log``.
+        harmonic_pressure (float): Harmonic contribution to the pressure.
+            If ommitted, the HMA pressure can still be computed, but will be
+            similar in precision to the conventional pressure.
```

Comment:
@joaander do you know of a simple link or some advice we could give it computing this quantity so users can use it.

Reply:
@ajschult could you help out here? 

If you haven't seen the announcements we are re-writing much of HOOMD's interface for a v3 release to make it more Pythonic. Along the way we are improving documentation and testing across the board. @danevans09 has ported the API for HMA to v3, but we could use your input on how to clarify the documentation. It would be great if you could help us develop a good validation test as well.